azext_iot/central/params.py

```diff
@@ -120,6 +120,12 @@ def load_central_arguments(self, _):
             help="Tenant ID for service principal to be added to the app. Object ID must also be specified. ",
         )
 
+        context.argument(
+            "token_id",
+            options_list=["--token-id", "--tkid"],
```

Comment:
Specifically for the short aliases of "-id" based arguments, as a guideline for other command groups (and trying to adhere moving forward), we want to try and make them single character where possible.

I'm not necessarily saying make the change here (and there are multiple -id args for this command group) but keep in mind the guideline. The ease of understanding and usability of args is high pri.